wot-security-metadata.md

```diff
@@ -306,9 +306,9 @@ HTTP authentication for HTTPS links and OCF access control lists for equivalent
 
 In this example, we have three different security configurations: HTTP Basic Authentication,
 OCF access control lists (the CoAP interface is actually to OCF devices), and an API key.
-We also have one case where _no_ security is used (the HTTPS interface to get overheating events)
+We also have one case where _no_ security is used (the HTTP-over-TLS interface to get overheating events)
 and another case where two are required (to turn on the light by CoAP/OCF, we need both
-access rights in the ACL _and_ an API key; the corresponding HTTPS interface needs both
+access rights in the ACL _and_ an API key; the corresponding HTTP-over-TLS interface needs both
```

Comment:
maybe we should do something like "HTTP-over-TLS interface (https link) needs..." and similarly later